### ISSUE TITLE
The `mime-guess` feature doesn't work

### ISSUE THREAD

**Issue by vicanso** [pyrossh/rust-embed#209]
When I enable the `mime-guess` feature like this:

```
rust-embed = { version = "6.6.0", features = ["compression", "mime-guess"] }
```
 I can't get the mimetype from metadata, and the build error is:

```
24 |         println!("{:?}", file.metadata.mimetype());
   |                                        ^^^^^^^^ method not found in `Metadata`
```

**Comment by BBaoVanC** [comment]
I'm confused too, it works in my app (https://github.com/BBaoVanC/bobashare, in bobashare-web/src/static_routes.rs), but I don't see Metadata::mimetype listed at all in the rustdoc (although maybe it can't tell it exists), and this simple code fails:

```rs
use rust_embed::RustEmbed;

#[derive(RustEmbed)]
#[folder = "static/"]
struct Asset;

fn main() {
    Asset::get("abc").map(|f| f.metadata.mimetype());
}
```

Not sure what is going on and how it's working fine for me

**Comment by vicanso** [comment]
@pyrossh  I find that if I add this dependency, it works. 

```
rust-embed-utils = { version = "7.5.0", features = ["mime-guess"] }
```

**Comment by pyrossh** [comment]
@vicanso thanks, that was it. The mime-guess feature didn't include the utils crate feature hence the issue. https://github.com/pyrossh/rust-embed/blob/master/Cargo.toml#L87. I'ave added it now and released v6.6.1 and it works.

Here is a simple example,
```rs
use rust_embed::RustEmbed;

#[derive(RustEmbed)]
#[folder = "static/"]
struct Asset;

fn main() {
    println!("{:?}", Asset::get("main.js").unwrap().metadata.mimetype())
}
```
